scripts/Start Stable Diffusion UI.cmd

```diff
@@ -1,8 +1,29 @@
 @echo off
 
 cd /d %~dp0
+echo Install dir: %~dp0
+
 set PATH=C:\Windows\System32;%PATH%
 
+if exist "on_sd_start.bat" (
+    echo ================================================================================
+    echo.
+    echo !!!! WARNING !!!!
+    echo.
+    echo It looks like you're trying to run the installation script from a source code 
+    echo download. This will not work.
+    echo.
+    echo Recommended: Please close this window and download the installer from
+    echo https://stable-diffusion-ui.github.io/docs/installation/
+    echo.
+    echo ================================================================================
+    echo.
+    echo Not Recommended: If you're sure that you want to run the installer from this
```

Comment:
This scenario shouldn't allow the user to continue. Because we really can't run the installer from inside the scripts folder (after checking out the repo). So I'd suggest exiting after letting the user press enter.

Reply:
Then maybe it's better to check if the .git directory exists? That way you would know that it's a git checkout.

Reply:
There are two scenarios. People using `git clone` and then there are people who download the zip file, which doesn't have the .git directory.
![image](https://user-images.githubusercontent.com/5852422/213435817-002875a8-56e4-4616-ae28-bd477b4a0b40.png)

Reply:
yeah, you are right, you can also download the zip... haven't thought about that.